skills/tedi-angular/SKILL.md

```
@@ -0,0 +1,176 @@
+
+Angular component library with 38+ accessible, standalone components. Built on Angular 19+ with signal-based APIs and design tokens from `@tedi-design-system/core`.
+
+## Installation
+
+```bash
+npm install @tedi-design-system/angular @tedi-design-system/core
+```
+
+### Peer Dependencies
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)